common/presenters/moves-by-vehicle.js

```diff
@@ -0,0 +1,77 @@
+const { addMinutes, formatISO, isToday, parseISO } = require('date-fns')
+const { isEmpty, get, groupBy, orderBy, sortBy } = require('lodash')
+
+const setMissingToLocation = require('../helpers/move/set-missing-to-location')
+
+const moveToCardComponent = require('./move-to-card-component')
+
+module.exports = function movesByVehicle(data, locationKey = 'to_location') {
+  const locations = []
+  const groupedByLocation = groupBy(data, 'meta.vehicle_registration')
+
+  Object.entries(groupedByLocation).forEach(([group, moves], index) => {
+    const vehicleRegistration = get(moves[0], 'meta.vehicle_registration')
+    const expectedTimeMetaKey =
+      locationKey === 'to_location'
+        ? 'expected_collection_time'
+        : 'expected_time_of_arrival'
+
+    // TODO: Remove dummy times below
+    moves = moves.map(move => {
+      if (index === 9) {
+        move.meta[expectedTimeMetaKey] = formatISO(addMinutes(new Date(), 1))
+      }
+
+      if (index === 2) {
+        move.meta[expectedTimeMetaKey] = formatISO(addMinutes(new Date(), 46))
+      }
+
+      if (index === 3) {
+        move.meta[expectedTimeMetaKey] = formatISO(addMinutes(new Date(), 35))
+      }
+
+      if (index === 4 || index === 6) {
+        move.meta[expectedTimeMetaKey] = formatISO(addMinutes(new Date(), 50))
+      }
+
+      if (index === 7 || index === 8) {
+        move.meta[expectedTimeMetaKey] = formatISO(addMinutes(new Date(), 120))
+      }
+
+      return move
+    })
+
+    const sortedByTime = sortBy(moves, `meta.${expectedTimeMetaKey}`)
+    const expectedTime = get(sortedByTime[0], `meta.${expectedTimeMetaKey}`)
+    const hasLeftCustody =
+      moves.filter(move => ['in_transit', 'completed'].includes(move.status))
+        .length === moves.length
```

Comment:
Would something like `every` work here? I think this is checking that every move is in these states. I think this could be:
`moves.every(move => ['in_transit', 'completed'].includes(move.status))` which returns a boolean